pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -260,6 +256,11 @@ func defaultPriorities() sets.String {
 
 		// Prioritizes nodes that marked with taint which pod can tolerate.
 		factory.RegisterPriorityFunction2("TaintTolerationPriority", priorities.ComputeTaintTolerationPriorityMap, priorities.ComputeTaintTolerationPriorityReduce, 1),
+
+		// ImageLocalityPriority prioritizes nodes based on locality of images requested by a pod. Nodes with larger size
+		// of already-installed packages required by the pod will be preferred over nodes with no already-installed
```

Comment:
I just feel this comment is too verbose, how about stop at "pod will be preferred."?

Reply:
Sure - let me rephrase it.